api/rest/restapi_test.go

```diff
@@ -512,7 +512,7 @@ func TestAPIAddFileEndpoint_StreamChannelsFalse(t *testing.T) {
 
 		makePostWithContentType(t, rest, shardURL, fullBody, mpContentType, &resp)
 		lastHash := resp[len(resp)-1]
-		if lastHash.Cid != test.ShardingDirBalancedRootCID {
+		if lastHash.Cid.String() != test.ShardingDirBalancedRootCID {
 			t.Error("Bad Cid after adding: ", lastHash.Cid)
```

Comment:
```suggestion
			t.Error("Bad Cid after adding: ", lastHash.Cid.String())
```

I think lastHash.Cid would print something like `{"Qm...."}`

Reply:
I don't think so. It would use `String()`.